assets/js/frontend/form.js

```diff
@@ -323,6 +323,8 @@ setTimeout(() => jQuery(function ($) {
     }
 
     function reloadWidget() {
+        const settings = window.wc.wcSettings.getSetting('power_board_data', {});
+        settings.selectedToken = ''
```

Comment:
@FabianaRBarros Out of interest, how does this fix it in terms of where does the "settings" get used?

Reply:
For context, when we select a saved card, we add its token to `settings.selectedToken` and when we try to submit the new card, there's a validation to see if we already have a `settings.selectedToken` - in the case we have a saved card selected - and if not, we wait for the widget to return this token. 
What was happening is that since we previously selected a card and did not reset its value from `settings.selectedToken`, it would assumed the saved card was still selected - causing the charge to be done on the previously selected card and not on the card that was used in the widget.

Reply:
@FabianaRBarros Lovely I get you, thanks